### ISSUE TITLE
Multiple redirect URIs support

### ISSUE THREAD

**Issue by TomUdr** [appvia/mock-oidc-user-server#4]
It would be nice if you add support for multiple redirect URIs.
CLIENT_REDIRECT_URI could handle e.g. comma separated values.

**Comment by jits** [comment]
Sounds like a potentially good idea, @TomUdr – I'd be curious to hear a use case for this, if you have one?

In theory, since this is designed purely for development and CI purposes, it should only need a single redirect URI.

**Comment by TomUdr** [comment]
We're using multi-tenant auth gateway which is handling multiple IDPs. For different tenants I would have to deploy different Mock OIDC servers with different CLIENT_REDIRECT_URIs. Which is of course doable, I'm just pointing to fact that underlaying https://github.com/panva/node-oidc-provider supports this.

**Comment by jits** [comment]
Yep, `node-oidc-provider` definitely supports this… however this wrapper is a fairly opinionated use of it.

That is not say we won't add/support this capability – just would like to understand the case better: with the handling of multiple IdPs the mock OIDC server would still redirect to just `localhost` (for local dev) or whatever host is setup in CI (in which case CI config could be different for it) – is there more to it than this?
